frontend/src/components/Collection/components/CollectionDatasetsGrid/components/Row/DownloadDataset/index.tsx

```diff
@@ -1,14 +1,10 @@
-import { Spinner } from "@blueprintjs/core";
-import * as React from "react";
-import { FC, useCallback } from "react";
+import React, { ElementType, FC, useState } from "react";
 import { useDatasetAssets } from "src/components/Collection/components/CollectionDatasetsGrid/components/Row/DownloadDataset/util";
 import Content from "src/components/Collections/components/Dataset/components/DownloadDataset/components/Content";
-import { StyledButton } from "src/components/Collections/components/Dataset/components/DownloadDataset/style";
-import Modal from "src/components/common/Modal";
-import { ModalContentWrapper } from "./style";
+import { Dialog } from "src/components/Datasets/components/DownloadDataset/style";
 
 interface Props {
-  Button?: React.ElementType;
```

Comment:
In this specific use case, there is no need for `Button` to be optionally defined, especially since it is already defined by default. Changing Button to required, we can eliminate the need for `StyledButton`, resulting in the removal of unused styles in `frontend/src/components/Collections/components/Dataset/components/DownloadDataset/style.ts` and `frontend/src/components/Collections/components/Dataset/common/style.ts`.

Reply:
Sounds good!